test/services/FootprintService.test.js

```diff
@@ -149,12 +149,7 @@ describe('api.services.FootprintService', () => {
         .then(role => {
           assert(role)
           assert(role.id)
-          return FootprintService.find('User', userId)
-        })
-        .then(user => {
-          assert(user[0])
-          assert.equal(user[0].roles.length, 1)
-          assert.equal(user[0].roles[0].name, 'createassociatedrole')
```

Comment:
Should this work? It should be user instead of user[0] from my testing, and in addition user.roles was never defined from what I could see.